test/test_utils.py

```diff
@@ -70,12 +81,16 @@ def test_non_spaces_location(self):
         shutil.rmtree(bad_path, ignore_errors=True)
 
     def test_set_path(self):
-        install_dir = os.path.expanduser(os.path.join('~', '.cmdstanpy'))
-        install_version = os.path.expanduser(
-            os.path.join(install_dir, get_latest_cmdstan(install_dir))
-        )
-        set_cmdstan_path(install_version)
-        self.assertEqual(install_version, cmdstan_path())
+        if 'CMDSTAN' in os.environ:
```

Comment:
If `CMDSTAN` is not set, then it should be a failure since if it is not set, utils.py should be setting it to the default location of cmdstan.